torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -617,11 +617,23 @@ void validateAlignedVectorizedFusionInputOutput(
   for (auto i = aten_tensor.ndimension() - 1; i >= 0; --i) {
     const auto stride = aten_tensor.strides().at(i);
     const auto size = aten_tensor.sizes().at(i);
+    auto root_id = tv->getMaybeRFactorDomain()[i];
+    const auto is_expanded_broadcasting =
+        root_id->isBroadcast() && root_id->hasExpandedExtent();
```

Comment:
Do we need to check for expanded broadcast here? I was under the impression that even a normal broadcast with an arbitrary stride should also pass the test, since we never index into broadcast dimension and it shouldn't break vectorization.

Reply:
Non-expanded broadcasting is covered by the `size == 1` check below. I keep "expanded broadcast" a separate case because I want to check that, if it is an expanded broadcast, then it needs to have stride 0. But for regular broadcast, there is no such limitation.